hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -237,19 +241,47 @@ public void stop() throws Exception {
     scheduler.shutdownNow();
   }
 
+  /**
+   * Find the OM leader's address to get the snapshot from.
+   */
+  @VisibleForTesting
+  public String getOzoneManagerSnapshotUrl() throws IOException {
+    if (!configuration.getBoolean(
+        OMConfigKeys.OZONE_OM_RATIS_ENABLE_KEY, false)) {
```

Comment:
For single node RATIS, we can still use the regular URL.

Reply:
OM AFAIK is either 3 node RATIS or no HA based on this configuration, although not sure why we cannot check, the result should be the same.